### DISCUSSION TITLE
Graph Tooltip, Single mode show all overlapping points

### DISCUSSION THREAD

**Discussion by MichaelTendoSsemwanga** [grafana/grafana#37223] [Dashboards, Panels & Visualizations]
My use case is Stacked area chart in Timeseries panel, 
![image](https://user-images.githubusercontent.com/24094521/127038332-e6c87f57-e643-4f65-86f8-b171e90143a6.png)
The null timeseries overlap on non-null time series and on hover, tooltip show no value, because of the null series.
Kindly consider this feature, I believe more users will find the utility.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
